adaptors task/student/src/impl.cpp

```diff
@@ -1,8 +1,32 @@
 #include "impl.h"
 #include <vector>
+#include <stack>
 
 const std::vector<std::pair<char, char>> brackets {
     {'(', ')'},
     {'[', ']'},
     {'{', '}'}
 };
+
+bool isValid(const std::string& source)
+{
+    std::stack<char> stackChecker;
+
+    for (auto nextSymbol: source) {
+        for (auto nextPair: brackets) {
+            if (nextSymbol == nextPair.first) {
```

Comment:
Please name this condition like
const bool isOpenBracket = nextSymbol == nextPair.first;